ddl-scripts/create_tables_mysql.sql

```diff
@@ -0,0 +1,95 @@
+CREATE TABLE IF NOT EXISTS event_journal(
```

Comment:
Is it possible to add a source header? this one - https://www.apache.org/legal/src-headers.html#headers - not the strange Pekko one mentioning Akka